README.md

```diff
@@ -98,6 +98,10 @@ Now `Client ID` and `Secret` must be set to `.env` as `KEYCLOAK_SYSTEM_USER_CLIE
 
 8. Go to Clients -> Find `badgerdoc-internal` -> Service Account Roles -> Client Roles -> master-realm -> Find `view-users` and `view-identity-providers` in Available Roles and add to Assigned Roles
 
+9. Go to Clients -> create roles: presenter, manager, role-annotator, annotator, engineer. Open admin role, go to Composite Roles -> Realm Roles and add all these roles 
+
+10. Set WEB_CORS=* in .env file, go to Clients -> badgerdoc-internal and set "Valid Redirect URIs" = * and "Web Origins" = *
```

Comment:
Let's set WEB_CORS=* to `.env.example` as default value and remove from documentation this step. I think `*` it's very common for local development